README.md

```diff
@@ -37,6 +38,15 @@ namaste
 dhanyabad
 </pre>
 
+## Input
+### `laga` keyword will be used as input statement
+<pre>
+namaste
+yo laga a;
```

Comment:
Please remove laga, other's seems fine

Reply:
I would love it if you make changes to this commit yourself. 
And one thing I want to know 😁
Do you remember me ? 😁